src/main/java/uk/gov/hmcts/divorce/caseworker/event/page/CorrectPaperCase.java

```diff
@@ -0,0 +1,195 @@
+package uk.gov.hmcts.divorce.caseworker.event.page;
+
+import uk.gov.hmcts.ccd.sdk.api.CaseDetails;
+import uk.gov.hmcts.ccd.sdk.api.callback.AboutToStartOrSubmitResponse;
+import uk.gov.hmcts.divorce.common.ccd.CcdPageConfiguration;
+import uk.gov.hmcts.divorce.common.ccd.PageBuilder;
+import uk.gov.hmcts.divorce.divorcecase.model.Applicant;
+import uk.gov.hmcts.divorce.divorcecase.model.Application;
+import uk.gov.hmcts.divorce.divorcecase.model.CaseData;
+import uk.gov.hmcts.divorce.divorcecase.model.HelpWithFees;
+import uk.gov.hmcts.divorce.divorcecase.model.Jurisdiction;
+import uk.gov.hmcts.divorce.divorcecase.model.MarriageDetails;
+import uk.gov.hmcts.divorce.divorcecase.model.Solicitor;
+import uk.gov.hmcts.divorce.divorcecase.model.State;
+
+import java.util.ArrayList;
+import java.util.List;
+import java.util.Objects;
+
+import static uk.gov.hmcts.divorce.divorcecase.validation.ValidationUtil.SOT_REQUIRED;
+
+public class CorrectPaperCase implements CcdPageConfiguration {
+
+    @Override
+    @SuppressWarnings("PMD.ExcessiveMethodLength")
+    public void addTo(PageBuilder pageBuilder) {
+        pageBuilder
+            .page("Correct paper case", this::midEvent)
```

Comment:
since you have only one page not sure MidEvent will be the right one ? Should we use about to submit ?
midevent is triggerred when you move from one page to another since you only. have one page might be good to use about to submit ?

Reply:
Sure, will move 👍